device/dell/x86_64-dellemc_z9332f_d1508-r0/DellEMC-Z9332f-M-O16C64/th3-z9332f-16x400G-64x100G.config.bcm

```diff
@@ -45,34 +45,94 @@ l2xlrn_thread_interval=50000
 l2xlrn_intr_en=0
 
 pbmp_xport_xe=0xffffffFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFE
-
 phy_an_c73=3
 
-portmap_1.0=1:400
-portmap_5.0=9:400
-portmap_10.0=17:400
-portmap_14.0=25:400
+portmap_20.0=33:100:2
+portmap_21.0=35:100:2
+portmap_22.0=37:100:2
+portmap_23.0=39:100:2
+
+portmap_24.0=41:100:2
+portmap_25.0=43:100:2
+portmap_26.0=45:100:2
+portmap_27.0=47:100:2
+
+portmap_28.0=49:100:2
+portmap_29.0=51:100:2
+portmap_30.0=53:100:2
+portmap_31.0=55:100:2
+
+portmap_33.0=57:100:2
+portmap_34.0=59:100:2
+portmap_35.0=61:100:2
+portmap_36.0=63:100:2
+
+portmap_40.0=65:100:2
+portmap_41.0=67:100:2
+portmap_42.0=69:100:2
+portmap_43.0=71:100:2
+
+portmap_44.0=73:100:2
+portmap_45.0=75:100:2
+portmap_46.0=77:100:2
+portmap_47.0=79:100:2
+
+portmap_49.0=81:100:2
+portmap_50.0=83:100:2
+portmap_51.0=85:100:2
+portmap_52.0=87:100:2
+
+portmap_53.0=89:100:2
+portmap_54.0=91:100:2
+portmap_55.0=93:100:2
+portmap_56.0=95:100:2
 
-portmap_20.0=33:400
-portmap_24.0=41:400
-portmap_29.0=49:400
-portmap_33.0=57:400
+portmap_1.0=1:100:2
+portmap_2.0=3:100:2
+portmap_3.0=5:100:2
+portmap_4.0=7:100:2
+
+portmap_5.0=9:100:2
+portmap_6.0=11:100:2
+portmap_7.0=13:100:2
+portmap_8.0=15:100:2
+
+portmap_10.0=17:100:2
+portmap_11.0=19:100:2
+portmap_12.0=21:100:2
+portmap_13.0=23:100:2
+
+portmap_14.0=25:100:2
+portmap_15.0=27:100:2
+portmap_16.0=29:100:2
+portmap_17.0=31:100:2
+
+portmap_80.0=129:100:2
+portmap_81.0=131:100:2
+portmap_82.0=133:100:2
+portmap_83.0=135:100:2
+
+portmap_84.0=137:100:2
+portmap_85.0=139:100:2
+portmap_86.0=141:100:2
+portmap_87.0=143:100:2
+
+portmap_89.0=145:100:2
+portmap_90.0=147:100:2
+portmap_91.0=149:100:2
+portmap_92.0=151:100:2
+
+portmap_93.0=153:100:2
+portmap_94.0=155:100:2
+portmap_95.0=157:100:2
+portmap_96.0=159:100:2
```

Comment:
Can you please explain how the portmap_* is mapped to front panel ports ? Does portmap_20 maps to Ethernet0 ? Is there any logic which define this mapping ?

Reply:
portmap_<logicalport>=hardware_port:bandwidth
dport_map_port_logicalport will help to order the Front end ports